openhcl/virt_mshv_vtl/src/processor/mod.rs

```diff
@@ -240,10 +241,8 @@ mod private {
 }
 
 pub struct BackingSharedParams<'a> {
-    #[cfg(guest_arch = "x86_64")]
-    pub(crate) cvm_state: Option<&'a crate::UhCvmPartitionState>,
-    #[cfg(not(guest_arch = "x86_64"))]
-    pub(crate) _phantom: &'a (),
+    pub(crate) _partition: &'a UhPartitionInner,
```

Comment:
Nothing is using the partition today, but I suspect that won't be the case for long.